modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -122,17 +118,21 @@ helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --crea
 --
 <1> Set the number of cores to an integer such as `4`. The more cores that are allocated to Redpanda, the more throughput it can support. For a description of CPU resource units, see the https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu[Kubernetes documentation^].
 +
-NOTE: If `resources.cpu.cores` is set using fractions, such as `1100m`, Redpanda may not use all the allocated CPU due to CFS quotas, leading to potential performance bottlenecks. When you use CPU fractions, <<pinning, CPU pinning>> is disabled.
-<2> This default setting allows each Redpanda thread to run on a dedicated core for maximum performance. This setting is recommended for environments where high throughput and low latency are critical, and sufficient hardware resources are available.
+NOTE: If you use fractional values, <<pinning, CPU pinning is disabled>>.
+<2> This default setting allows each Redpanda thread to run on a dedicated core for maximum performance.
 --
 
 . <<qos, Enable the `Guaranteed` QoS class for your Pods that run Redpanda>>.
++
+Only containers that are both part of a `Guaranteed` Pod and have integer CPU requests are assigned exclusive CPUs.
 
 [[pinning]]
 === Disable pinning in shared environments
 
 On a dedicated worker node, CPU pinning allows maximum throughput, but in a development environment, where you may run other Pods on the same worker node as Redpanda, CPU pinning can cause your other processes to run slower than usual. In these shared environments, you can enable overprovisioning so that Redpanda reduces the aggressiveness of CPU usage to prevent resource hogging and ensuring fair CPU time distribution among all processes.
 
+CAUTION: When `resources.cpu.cores` is set to a fractional value, such as `1100m`, Redpanda may not use all the allocated CPU due to CFS quotas, leading to potential performance bottlenecks.
```

Comment:
I feel this doesn't convey the core issue just the side effect. Putting on my user hat I feel that "CFS quotas" is suddenly being lobbed at me and I likely won't understand why CFS is a problem in this content.

```suggestion
CAUTION: When `resources.cpu.cores` is set to a fractional value, such as `1100m`, the Redpanda Pod is not eligible for exclusive core access which will lead to potential performance bottlenecks.
```